internal/controller/processor.go

```diff
@@ -201,17 +199,26 @@ func (processor *AppConfigurationProviderProcessor) processSecretReferenceRefres
 
 	// Only resolve the secret references that not specified the secret version
 	secretReferencesToSolve := make(map[string]*loader.TargetSecretReference)
+	cachedSecretReferences := make(map[string]*loader.TargetSecretReference)
```

Comment:
This name is really confusion-leading, what you do here is to deep copy the `reconcileState.ExistingSecretReferences`, right?

Reply:
yes, how about `copiedSecretReferences`?